metrics/api/src/main/java/io/helidon/metrics/api/MetadataBuilder.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+/**
+ * Convenience class for easing the migration to the Helidon metrics API.
+ * @deprecated Use the various meter builders to prepare their characteristics, for example
+ * {@link io.helidon.metrics.api.Gauge.Builder}
+ */
+@Deprecated(since = "4.0.0")
+public class MetadataBuilder {
```

Comment:
I think the whole `Metadata` should be changed to `MetadataBlueprint` and generate the type, that will make it aligned with our API guidelines and reduce code.

Reply:
see above

Reply:
The type is gone.